README.md

```diff
@@ -1 +1,137 @@
-# Budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Budget App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Version](#live-version)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budget App** is a web project based on Ruby on Rails, using Postgresql as a database. To use this app, the user has to create an account, then log in, at this point the user can manage his data like creating and displaying new categories, then new records for their trades(transactions). Each category has a name and an icon, and each trade(transaction) has a name, amount, and category. The user also can see for each category the total amount and the total amount of trades.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **The page is built with Ruby on Rails**
+- **The desing is for mobile first approach**
+- **For handling user's log and register the App uses device gem**
+
+### Live demo <a name="live-demo">
+[Budget App](https://capstone-budgetapp.onrender.com)
+</a>
+
+### Presentation Video <a name="presentation">
+[Budget App](https://www.loom.com/share/231cd971f0094169b35112f8a25abbb2?sid=ba4041f9-58dd-42f3-a85f-18730283a6c8)
+</a>
```

Comment:
- [ ] Kindly please indicate the piece of code or something that you find interesting in the process of building the app in your video presentation as well. 👍 

<img width="551" alt="Untitled" src="https://github.com/MenendezON/Budget-app/assets/103386183/ff71b991-6640-4b2a-a5e0-33621bfc05e1">